### DISCUSSION TITLE
16. What open source models are available for businesses?

### DISCUSSION THREAD

**Discussion by MichaelRimler** [phuse-org/OSTCDA#16] [General]
What open source models are available for businesses?

**Comment by KatjaGlassConsulting** [comment]
Good overview available here: https://en.wikipedia.org/wiki/Business_models_for_open-source_software

- Professional services
- Branded merchandise
- Software as a service
- Voluntary donations
- Crowdsourcing
- Training and certification
- Partnership with funding organizations
- Advertising-supported software
- Bounty driven development
- Pre-order/crowdfunding/reverse-bounty model
- Selling optional proprietary extensions

**Comment by KatjaGlassConsulting** [comment]
Examples available in our industry:
- Company projects - Admiral, Shiny Apps, ... - companies allow their employees to invest their working time
- RPact - membership funds project, some services only for members, source code is open source
- R Consortium - funding some OS projects / supporting events
- Atorus - deliver OS, provide services and use partnerships
- no example - Pinnacle Community Edition, this is not using a FOSS license (even though you can review the code, you are not allowed to reuse and similar)
